config/config.go

```diff
@@ -132,13 +108,9 @@ func (ctx *GlobalOpts) CleanupTmp() {
 
 // OutputDir parses a filepath based on GlobalOpts.InstallDir and the datetime this was initialized
 func (ctx *GlobalOpts) OutputDir() string {
-	year, month, day := ctx.StartTime.Date()
-	hour, min, sec := ctx.StartTime.Clock()
-	yearMonthDay := fmt.Sprintf("%04d%v%02d", year, month, day)
-	timestamp := fmt.Sprintf("%02d%02d%02d", hour, min, sec)
 	execName := utils.GetExecutableName()
 
-	base := filepath.Join(ctx.InstallDir, "output", yearMonthDay, timestamp, execName)
```

Comment:
Removed the timestamp output feature (1) for simplicity and (2) to reduce our bug surface area, and additionally (3) because everyone keeps overriding it anyhow